src/lib/paths.ts

```diff
@@ -30,6 +30,16 @@ export const studyDocumentURL = (studyId: string, type: StudyDocumentType, fileN
 
 export const studyCodeURL = (jobId: string, fileName: string) => `/dl/study-code/${jobId}/${fileName}`
 
+export const coderUserInfoPath = (username: string) => `/api/v2/users/${username}`
+export const coderUsersPath = () => `/api/v2/users`
+export const coderWorkspaceCreatePath = (organization: string, username: string) =>
```

Comment:
I thought we had decided against using organizations for the time being and instead use [this endpoint](https://coder.com/docs/reference/api/workspaces#create-user-workspace) :man_shrugging:

Reply:
When no organization was specified when creating users,  the calls returns a 400 error. But by using the default organization, it works successfuly so instead of creating organization we can use the one that is created when coder is deployed.

Reply:
I see. Is there a static value or something for the "default organization"? 

I'll also see if I can better understand the `400` behavior. From a quick test, when I create a workspace in the UI in QA I see a `POST` to `https://coder.qa.safeinsights.org/api/v2/users/71dab09f-0502-446e-b16d-64635c8ccc8c/workspaces` :man_shrugging: .

Reply:
I used the organization endpoint using curl and retrieved the default org id. 
```
curl -X GET https://coder.sandbox.safeinsights.org/api/v2/organizations -H 'Accept: application/json' -H 'Coder-Session-Token: ....'
[{"id":"fcb3aeae-1c1c-4b9b-98d0-50114805691d","name":"coder","display_name":"Coder","icon":"","description":"Builtin default organization.","created_at":"2025-10-03T15:48:58.034Z","updated_at":"2025-10-03T15:48:58.034Z","is_default":true}]
```